internal/shared/db/model/setup.go

```diff
@@ -69,6 +72,7 @@ var collections = map[string][]index{
 		{Indexes: map[string]int{"active_tvl": -1}, Unique: false},
 		{Indexes: map[string]int{"active_delegations": 1}, Unique: false},
 	},
+	V2FinalityProvidersLogosCollection: {{Indexes: map[string]int{}}},
```

Comment:
that's another thing that I wanted to discuss:
- since generally we don't want to touch indexer collections from another service we should use only api collections
- existing collection related to finality provider is `v2_finality_providers_stats`, but I didn't want to put something realted to UI inside *stats collection

So I created a new one, maybe we should make it generic "logos" where we will store fp logo and bsn logo at the same time, wdyt?

Reply:
Changed the name to `v2_finality_providers_metadata`